modules/task/migrate.go

```diff
@@ -58,6 +58,9 @@ func runMigrateTask(t *models.Task) (err error) {
 		t.EndTime = timeutil.TimeStampNow()
 		t.Status = structs.TaskStatusFailed
 		t.Message = err.Error()
+
```

Comment:
```suggestion
	// Reloads the repo before we zero out repo's ID, this is to prevent
	// that the repo cannot be deleted.
```

Let's add a comment for such "unusual" line of code.

Reply:
```suggestion
	// Ensure that the repo loaded before we zero out the repo ID from the task - thus ensuring that we can delete it
```